deployments/kubernetes/install/scripts/install-cni.sh

```diff
@@ -32,6 +32,25 @@ MOUNTED_CNI_NET_DIR=${MOUNTED_CNI_NET_DIR:-/host/etc/cni/net.d}
 CNI_CONF_NAME=${CNI_CONF_NAME:-$(ls ${MOUNTED_CNI_NET_DIR} | head -n 1)}
 CNI_CONF_NAME=${CNI_CONF_NAME:-10-calico.conflist}
 KUBECFG_FILE_NAME=${KUBECFG_FILE_NAME:-ZZZ-istio-cni-kubeconfig}
+CFGCHECK_INTERVAL=${CFGCHECK_INTERVAL:-1}
```

Comment:
I think a default of `10` would be better for `sleep`, just like before.

Reply:
I am thinking we need a smaller default to shorten the window of misconfig for the cases highlighted by issue #82 and Spike's comment here:  https://github.com/istio/istio/issues/11037#issuecomment-460368114

The gist is that we can't ensure the init order of the pods that do the per-node CNI installations (like istio-cni-node) when adding new nodes to the cluster AND we can't ensure control of when user pods start getting scheduled to the newly added nodes.

Reply:
Got it.